aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/AliyunOSSMockLocalStore.java

```diff
@@ -170,17 +174,21 @@ ObjectMetadata putObject(
 
   void deleteObject(String bucketName, String filename) {
     File bucketDir = new File(root, bucketName);
-    assert bucketDir.exists();
```

Comment:
in general, do we allow `assert`?

Reply:
nope, there's no real advantange in using the `assert` statement. Additionally, they aren't enabled so these checks have actually never be performed. We might actually want to add a checkstyle rule to prevent usage of `assert` in Java code (I'll open a separate PR for that)

Reply:
thanks. that's exactly where i was getting at with the question